src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/time.html.twig

```diff
@@ -14,9 +14,10 @@
     } %}
 {% endif %}
 
-{% set has_time_events = collector.events|length > 0 %}
 
 {% block toolbar %}
+    {% set has_time_events = collector.events|length > 0 %}
```

Comment:
shouldn't it be duplicated in `{% block panel %}` too?

Reply:
Yes that's right!